lua/telescope/builtin/internal.lua

```diff
@@ -926,6 +926,30 @@ internal.tagstack = function(opts)
     }):find()
 end
 
+internal.jumplist = function(opts)
+  opts = opts or {}
+  local jumplist = vim.api.nvim_eval('getjumplist()')[1]
+
+  for _, value in ipairs(jumplist) do
+    value.text = ''
+  end
+
+  -- reverse the list
+  local sorted_jumplist = {}
+  for i = #jumplist, 1, -1 do
+    sorted_jumplist[#sorted_jumplist+1] = jumplist[i]
```

Comment:
I believe you should be able to replace the line in the for loop with `table.insert(sorted_jumplist, jumplist[i])` if you'd like. `table.insert` is a part of Lua's table library which inserts an item into a table. If you only give two arguments to it, it will accept the table to insert into and the item you want to insert, inserting that item at the end of the table (similar to the `append()` method in Python for example). If you give 3 args, it would take the table to insert into, the position in the table to insert the item, and then the item to insert. Just a little bit easier to read I think using `table.insert` instead :smiley: